frontends/main/src/app-pages/DashboardPage/DashboardPage.tsx

```diff
@@ -277,38 +277,65 @@ const TabLabels = {
   [SETTINGS]: "Settings",
 }
 
-interface UserMenuTabProps {
+const DesktopTabLabel: React.FC<{
   icon: React.ReactNode
-  text: string
-  tabKey: string
-  value: string
-  currentValue: string
-  onClick?: () => void
-}
-
-const UserMenuTab: React.FC<UserMenuTabProps> = (props) => {
```

Comment:
The main issue was that, apparently, if you wrap the MUI tab component in your own custom component, you need to spread props, so something like:
```tsx
const UserMenuTab = ({ icon, text, tabKey, value, currentValue, onClick , ...others }) => {
    /* What we have now */
    return ( <Tab /*What we have now*/ {...others} />)
} 
```
would've worked.

But I just removed our custom wrapper and did a little refactoring.

See https://github.com/mui/material-ui/issues/27947#issuecomment-905318861